### PR TITLE
Added the client to the project

### PR DESCRIPTION
**Pull request by TannerHolladay**

### CODE REVIEW

File: client/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
```
Comment (kylehovey) [inline client/.gitignore:6]: Should this be ignoring a `.ts` file now?

File: client/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
```
Comment (kylehovey) [inline client/src/App.tsx:5]: Add React functional component type here

File: client/src/stories/Button.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+
+import { Button, ButtonProps } from './Button';
+
+export default {
```
Comment (kylehovey) [inline client/src/stories/Button.stories.tsx:6]: Add type for default export

Comment (kylehovey) [reply client/src/stories/Button.stories.tsx:6]: Never mind, I saw the `as Meta`

File: client/src/stories/Button.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  title: 'Example/Button',
+  component: Button,
+  argTypes: {
+    backgroundColor: { control: 'color' },
+  },
+} as Meta;
+
+const Template: Story<ButtonProps> = (args) => <Button {...args} />;
+
+export const Primary = Template.bind({});
```
Comment (kylehovey) [inline client/src/stories/Button.stories.tsx:16]: We should probably add types for these exports.

File: client/src/stories/Header.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+import { Header, HeaderProps } from './Header';
+
+export default {
+  title: 'Example/Header',
+  component: Header,
+} as Meta;
+
+const Template: Story<HeaderProps> = (args) => <Header {...args} />;
+
+export const LoggedIn = Template.bind({});
```
Comment (kylehovey) [inline client/src/stories/Header.stories.tsx:13]: We should add types for these exports too

File: client/.gitignore (hunk 1 above)
Comment (TannerHolladay) [reply client/.gitignore:6]: Why should they be ignored?

File: client/src/App.tsx (hunk 2 above)
Comment (TannerHolladay) [reply client/src/App.tsx:5]: What do you mean?

**Review by kylehovey — APPROVED** [review]
Noice!

**Pull request merged**